### PR TITLE
Fail fast on project creation

### PR DESCRIPTION
**Pull request by helgridly**
Should identify misconfigured tests, where the user doesn't actually have permissions to create projects but is trying to anyway.

Completely untested.

**PR checklist**
- [ ] For each library you've modified here, decide whether it requires a major, minor, or no version bump. (Click [here](/broadinstitute/workbench-libs/blob/develop/CONTRIBUTING.md) for guidance)

If you're doing a **major** or **minor** version bump to any libraries:
- [ ] Bump the version in `project/Settings.scala` `createVersion()`
- [ ] Update `CHANGELOG.md` for those libraries
- [ ] I promise I used `@deprecated` instead of deleting code where possible

In all cases:
- [ ] Get two thumbsworth of PR review
- [ ] Verify all tests go green (CI _and_ coverage tests)
- [ ] Squash commits and **merge to develop**
- [ ] Delete branch after merge

After merging, _even if you haven't bumped the version_: 
- [ ] Update `README.md` and the `CHANGELOG.md` for any libs you changed with the new dependency string. The git hash is the same short version you see in GitHub (i.e. seven characters). You can commit these changes straight to develop.

### CODE REVIEW

File: serviceTest/src/test/scala/org/broadinstitute/dsde/workbench/service/Google.scala [hunk 1]
```diff
@@ -19,5 +21,14 @@ object Google extends FireCloudClient with LazyLogging {
       logger.info(s"Getting billing account associated with $projectName")
       parseResponseAs[Map[String, String]](getRequest(s"https://content-cloudbilling.googleapis.com/v1/projects/$projectName/billingInfo?fields=billingAccountName")).get("billingAccountName")
     }
+
+    def canCreateBillingProjects(billingAccountId: String)(implicit token: AuthToken): Boolean = {
+      logger.info(s"Checking to see if user can create projects in account $billingAccountId")
+      //user has permissions if we get a 200 back
+      Try {
+        postRequest(s"https://cloudbilling.googleapis.com/v1/billingAccounts/01A82E-CA8A14-367457:testIamPermissions",
```
Comment (jmthibault79) [inline serviceTest/src/test/scala/org/broadinstitute/dsde/workbench/service/Google.scala:29]: oh this is handy

**Review by jmthibault79 — APPROVED** [review]
Seems legit.  Should cause a whole bunch of tests to fail, which we can pick off piecemeal.

**Comment by helgridly** [comment]
@jmthibault79 the plan was to NOT merge this and instead fix 'em all up in a branch, right? In which case you can use whatever -SNAP that Travis builds and piecemeal away.

**Comment by jmthibault79** [comment]
yes

**Comment by Qi77Qi** [comment]
can we close this now or @jmthibault79 would you like to take it over?

**Comment by jmthibault79** [comment]
I should not have ownership of this, so go ahead with whatever is appropriate.  I'm guessing this is very out-of-date at this point.

**Comment by helgridly** [comment]
I'm deeply offended that you're going to close my PR. 😜

**Comment by Qi77Qi** [comment]
@helgridly oh hii, wanna finish this up so that I won't be deeply offending you? 😆

**Comment by helgridly** [comment]
think of it as a gift from me to you. something to remember me by 💛

**Comment by Qi77Qi** [comment]
@helgridly ha...fine..guess I'll have to take your gift

**Pull request closed without merging**